src/plugins/orangehrmPimPlugin/Api/EmployeeAttachmentAPI.php

```diff
@@ -322,6 +464,13 @@ public function getValidationRuleForUpdate(): ParamRuleCollection
     }
 
     /**
+     * @OA\Delete(
+     *     path="/api/v2/pim/employees/{empNumber}/screen/{screen}/attachments",
+     *     tags={"Pim/Employee Attachment"},
```

Comment:
missing `@OA\PathParameter`
```suggestion
     *     tags={"Pim/Employee Attachment"},
     *     @OA\PathParameter(
     *         name="empNumber",
     *         @OA\Schema(type="integer")
     *     ),
     *     @OA\PathParameter(
     *         name="screen",
     *         @OA\Schema(type="string", enum=OrangeHRM\Entity\EmployeeAttachment::SCREENS)
     *     ),
```